src/main/java/it/gov/pagopa/payhub/activities/dto/massivedp/InstallmentIngestionFlowFileResult.java

```diff
@@ -0,0 +1,22 @@
+package it.gov.pagopa.payhub.activities.dto.massivedp;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.util.Map;
+
+/**
+ * DTO for the InstallmentIngestionResult, representing the result of installment file processing.
+ */
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+public class InstallmentIngestionFlowFileResult {
+    /** Map of IUDs and their corresponding Installments IDs */
+    private Map<String, String> iud2InstallmentsIdMap;
```

Comment:
This activity has not result to be retrieved (we should instead return the total number of rows and the number of row correctly handled)